server/apiRoutes/index.js

```diff
@@ -1,16 +1,19 @@
 const express = require('express');
 
 const apiRoutes = express.Router();
-// import api routes here
+//import api routes here
 
 const tradeChoreRoutes = require('./tradeChoreRoutes');
 const swapChoreRoutes = require('./swapChoreRoutes');
 const transferChoreRoutes = require('./transferChoreRoutes');
-const choresRoutes = require('./ChoresRoutes');
+const adminRoutes = require('./adminRoutes');
+const choresRoutes = require('./choresRoutes');
```

Comment:
you forgot to rename this file.

Reply:
Tried repushing to force these changes through again